zeppelin-interpreter/pom.xml

```diff
@@ -36,6 +36,14 @@
   <url>http://zeppelin.incubator.apache.org</url>
 
   <dependencies>
+
+    <dependency>
+      <groupId>org.apache.zeppelin</groupId>
+      <artifactId>zeppelin-spark-utils</artifactId>
```

Comment:
The only reason I added a new Maven **module** is that the code of display utility function is written in Scala and I did not want to _pollute_ existing Java code in the **interpreter** or **spark** module.

But anyway, it's very easy to change. Thanks to the **maven-scala-plugin**, the Scala code will be taken into account in the build process.

So just tell me in which project I should put the Scala code and I'll update my pull-request